### ISSUE TITLE
Add Colab about how to convert TFLite compatible seq2seq model

### ISSUE THREAD

**Issue by jaeyoo** [tensorflow/addons#2003]
**Describe the feature and the current behavior/state.**

**Relevant information**
- Are you willing to contribute it (yes/no): yes
- Are you willing to maintain it going forward? (yes/no): yes
- Is there a relevant academic paper? (if so, where): no
- Is there already an implementation in another framework? (if so, where): don't know
- Was it part of tf.contrib? (if so, where): no

**Which API type would this fall under (layer, metric, optimizer, etc.)**
seq2seq

**Who will benefit with this feature?**
Everyone who wants to use RNN models for NLP, TTS etc

**Any other info.**
This issue will have link to the colab later (WIP)

**Comment by dathudeptrai** [comment]
this is really helpful, i'm working on Seq2Seq based-on tf-addon and i want tf_addons has official tutorial about convert TFLite for seq2seq module. Pls take a look @guillaumekln @seanpmorgan

**Comment by jaeyoo** [comment]
https://github.com/tensorflow/addons/pull/1964

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
